apps/web/src/app/(private)/evaluations/(evaluation)/layout.tsx

```diff
@@ -1,38 +0,0 @@
-import { ReactNode } from 'react'
-
-import { Text } from '@latitude-data/web-ui'
-import BreadcrumpLink from '$/components/BreadcrumpLink'
-import { AppLayout } from '$/components/layouts'
-import { getCurrentUser } from '$/services/auth/getCurrentUser'
-import { ROUTES } from '$/services/routes'
-
-import { NAV_LINKS } from '../../_lib/constants'
-
-export default async function EvaluationLayout({
-  children,
-}: {
-  children: ReactNode
-}) {
-  const session = await getCurrentUser()
-  return (
-    <AppLayout
-      navigationLinks={NAV_LINKS}
-      currentUser={session.user}
-      breadcrumbs={[
-        {
-          name: (
-            <BreadcrumpLink
-              href={ROUTES.dashboard.root}
-              name={session.workspace.name}
-            />
-          ),
-        },
-        {
-          name: <Text.H5M>Evaluations</Text.H5M>,
-        },
-      ]}
-    >
-      {children}
-    </AppLayout>
-  )
-}
```

Comment:
is this breaking any modals?

Reply:
I think it's not
![image](https://github.com/user-attachments/assets/c11a5097-6315-4fee-a0ea-c73cdc5c13b9)

Reply:
Great catch 💯 but remove evaluation is already wrong in production

The button is in `/evaluations` but it goes to `/evaluations/:id/dashboard/destroy` which super weird weird
![image](https://github.com/user-attachments/assets/17fafe20-8d5a-4fc9-8f2a-4a68b4bb9503)

Reply:
why? doesn't look weird to me at all

Reply:
Before from `/evaluations` -> `evaluations/:id/destroy`

Now it goes `/evaluations` -> `/evaluations/:id/destroy`

In the same layout so modal is over evaluations list
![image](https://github.com/user-attachments/assets/23b539b3-4251-4686-9702-0dda0ea6a22d)